catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/GmtiMetacardType.java

```diff
@@ -58,5 +59,6 @@ public void initDescriptors() {
         descriptors.addAll(getDescriptors(CsdidaAttribute.getAttributes()));
         descriptors.addAll(getDescriptors(CsexraAttribute.getAttributes()));
         descriptors.addAll(getDescriptors(HistoaAttribute.getAttributes()));
+        descriptors.addAll(getDescriptors(PiatgbAttribute.getAttributes()));
```

Comment:
It would be nice to add all TRE attributes to a base metacard type instead of to both the Image and Gmti metacard types, but I do not know of a way to do that.